refinery/tool_manager/tests.py

```diff
@@ -637,6 +636,39 @@ def test_workflow_list_not_enough_inputs(self):
             )
             self.assertEqual(ToolDefinition.objects.count(), 0)
 
+    def test_visualization_annotation_extra_dirs_valid(self):
+        with open(
+            "{}/visualizations/LIST_visualization_good_extra_directories.json"
+            .format(TEST_DATA_PATH)
```

Comment:
We have `workflows` under `TEST_DATA_PATH` as well so I made the distinction a directory above.  
I have it on my list to clean up these long test strings (although they are quite descriptive)